coderd/conversion/provisionerjob.go

```diff
@@ -0,0 +1,33 @@
+package conversion
```

Comment:
I'd push back on having these all be separated files instead of one large `conversion.go`.

We already have like three other files named `provisionerjob.go` or some slight variant, so having another could be confusing.

Reply:
I don't mind consolidating.